<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageTime</class>
 <widget class="QWidget" name="WizardPageTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>32</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_zone">
       <property name="text">
        <string>Time Zone</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="comboBox_zone"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_daylight">
       <property name="text">
        <string>Daylight Saving Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="comboBox_daylight"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_synchronize">
       <property name="text">
        <string>Synchronize with NTP server</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_ntpServer">
       <property name="text">
        <string>NTP Server</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="MyLineEdit" name="lineEdit_ntpServer"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_manually">
       <property name="text">
        <string>Set Date and Time Manually</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_dateTime">
       <property name="text">
        <string>Set Date and Time</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="MyDateTimeEdit" name="dateTimeEdit">
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>mydatetimeedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
